### PR TITLE
[INV-73] Image Upload Functionality

### PR DESCRIPTION
**Pull request by muhammadjon8**
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,8 @@ WORKDIR /app
 
 COPY package*.json ./
 
-RUN npm ci
+RUN npm config set registry https://registry.npmjs.org/
```
Comment (NegatiV-64) [inline Dockerfile:10]: Why to use this command?

File: Dockerfile [hunk 2]
```diff
@@ -7,7 +7,8 @@ WORKDIR /app
...
+RUN npm install
```
Comment (NegatiV-64) [inline Dockerfile:11]: Why to replace ci?

File: docker-compose.yml [hunk 3]
```diff
@@ -24,31 +24,54 @@ services:
...
+      app-network:
     command: >
       sh -c "npx typeorm migration:run -d dist/typeorm.config.js && npm run start:prod"
+
   db:
     container_name: db
     image: postgres:17
     ports:
-      - '5432:5432'
+      - '15432:5432'
```
Comment (NegatiV-64) [inline docker-compose.yml:42]: Not sure that we need to map the outer port

File: docker-compose.yml [hunk 4]
```diff
@@ -24,31 +24,54 @@ services:
...
+      CORS_ENABLED: ${CORS_ENABLED}
       CORS_ALLOWED_ORIGINS: ${CORS_ALLOWED_ORIGINS}
-      CORS_ALLOWED_METHODS: ${COOKIES_REFRESH_MS}
+      CORS_ALLOWED_METHODS: ${CORS_ALLOWED_METHODS}
     depends_on:
       - db
+      - minio
     networks:
-      - app-network
+      app-network:
```
Comment (NegatiV-64) [inline docker-compose.yml:34]: Why it was changed?

**Review by NegatiV-64 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 4 above)
Comment (Savesh1508) [reply docker-compose.yml:34]: About environment variables - it's correct changes. We fixed my fault.

File: Dockerfile (hunk 1 above)
Comment (NegatiV-64) [reply Dockerfile:10]: Npm ci is a special command for running for such case6

File: docker-compose.yml (hunk 3 above)
Comment (muhammadjon8) [reply docker-compose.yml:42]: fixed

File: src/modules/uploads/minio.service.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Client } from 'minio';
+import { EnvConfig } from '../../shared/configs/env.config';
```
Comment (Savesh1508) [inline src/modules/uploads/minio.service.ts:4]: Use alias ( # )

File: Dockerfile [hunk 6]
```diff
@@ -7,6 +7,8 @@ WORKDIR /app
 
 COPY package*.json ./
 
+RUN npm config set registry https://registry.npmjs.org/
+RUN npm install
 RUN npm ci
```
Comment (Savesh1508) [inline Dockerfile:12]: Now you use both of npm ci and npm install?
You can delete npm install

**Pull request merged**